agents1/OfficialAgent.py

```diff
@@ -89,12 +100,17 @@ def decide_on_actions(self, state):
             for member in self._teamMembers:
                 if mssg.from_id == member and mssg.content not in self._receivedMessages:
                     self._receivedMessages.append(mssg.content)
-        # Process messages from team members
-        self._processMessages(state, self._teamMembers, self._condition)
+
         # Initialize and update trust beliefs for team members
         trustBeliefs = self._loadBelief(self._teamMembers, self._folder)
         self._trustBelief(self._teamMembers, trustBeliefs, self._folder, self._receivedMessages)
 
+        ## Alex: For some reason this is a list, but I printed it and it is always just the one human.
+        trustBelief = trustBeliefs[self._teamMembers[0]]
```

Comment:
yeah this is because it takes into account the fact that you can have multiple iterations of the game with humans with different names, so if you use the same name twice it will load the same trust values as the previous iteration. So you should access it with the human name instead like this:
```suggestion
        trustBelief = trustBeliefs[self._humanName]
```